docs/content/design/indexing-service.md

```diff
@@ -68,6 +68,12 @@ The overlord console can be used to view pending tasks, running tasks, available
 http://<OVERLORD_IP>:<port>/console.html
 ```
 
+#### Blacklisted Workers
+If the workers fail tasks above a threshold, the overlord will blacklist these workers. No more than 20% of the nodes can be blacklisted.
```

Comment:
Can we reference one of the config parameters here to make it a bit easier to track down the set of params on the config page (maybe even link to the config page).